test/microbench/org/apache/cassandra/test/microbench/ZeroCopyStreamingBenchmark.java

```diff
@@ -40,12 +40,7 @@
 import org.apache.cassandra.db.RowUpdateBuilder;
 import org.apache.cassandra.db.commitlog.CommitLog;
 import org.apache.cassandra.db.compaction.CompactionManager;
-import org.apache.cassandra.db.streaming.CassandraEntireSSTableStreamReader;
-import org.apache.cassandra.db.streaming.CassandraEntireSSTableStreamWriter;
-import org.apache.cassandra.db.streaming.CassandraOutgoingFile;
-import org.apache.cassandra.db.streaming.CassandraStreamHeader;
-import org.apache.cassandra.db.streaming.CassandraStreamReader;
-import org.apache.cassandra.db.streaming.CassandraStreamWriter;
+import org.apache.cassandra.db.streaming.*;
```

Comment:
nit: I think we generally try to avoid wildcards.

Reply:
+1